### PR TITLE
#257 Doc short link

### PR DESCRIPTION
**Pull request by pinaf**
For #257
- added UI short link and wired it to `Doc.shortUrl`

### CODE REVIEW

**Comment by karato** [comment]
@pinaf Many thanks for the PR, let me find a reviewer for it

**Comment by karato** [comment]
@darkled review it

**Comment by asaen** [comment]
@rultor merge

File: src/main/java/com/nerodesk/takes/TkDocs.java [hunk 1]
```diff
@@ -138,6 +138,7 @@ private static XeSource source(final Doc doc, final String name,
                         .add("name").set(name)
                 ),
                 new XeLink("read", home.path("read")),
+                new XeLink("shortlink", new Href(doc.shortUrl())),
```
Comment (yegor256) [inline src/main/java/com/nerodesk/takes/TkDocs.java]: why not just "short"?

**Comment by yegor256** [comment]
@pinaf what about a unit test??

**Comment by pinaf** [comment]
@yegor256 fixed. please take a look.

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by karato** [comment]
@darkled Much appreciated! **16 mins** was added to your account, payment ID `56771639`, time spent is 73 hours and 55 mins... extra minutes for review comments (c=1)... added +16 to your rating, now it is equal to [+2095](http://www.netbout.com/b/35091?open=rating)

**Comment by karato** [comment]
@rultor deploy

**Pull request merged**